package/rotating_zmq_logger/src/rotating_logger.cc

```diff
@@ -38,15 +38,19 @@
 static const std::string LOG_SUFFIX = ".sbp";
 static const size_t LOG_NAME_LEN = 4 + 1 + 5 + 4;
 
-void RotatingLogger::debug_printf(const char* msg, ...) {
-  if (!_verbose_logging) {
-    return;
+static const int LOG_EMERG = 0;  /* system is unusable */
```

Comment:
These should be defined in `<syslog.h>` which you can get from `<libpiksi/logging.h>`